### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by sadelcode**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,6 @@
 
     </head>
     <body>
-        
+        <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (RashedCode) [inline index.html]: Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?

Comment (sadelcode) [reply index.html]: Sure, can do!

**Review by RashedCode — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by RashedCode — APPROVED** [review]
Well done Saif!

**Pull request merged**